spacy/__init__.py

```diff
@@ -42,14 +44,24 @@ def load(
     disable (Iterable[str]): Names of pipeline components to disable. Disabled
         pipes will be loaded but they won't be run unless you explicitly
         enable them by calling nlp.enable_pipe.
+        enable (Iterable[str]): Names of pipeline components to enable. All other
+            pipes will be disabled (and can be enabled using `nlp.enable_pipe`).
     exclude (Iterable[str]): Names of pipeline components to exclude. Excluded
         components won't be loaded.
+    include (Iterable[str]): Names of pipeline components to include. All other
+        components will be excluded.
```

Comment:
I think it's slightly more clear to say "All other components will not be loaded".